db/migrate/20250318024953_copy_solved_topic_custom_field_to_discourse_solved_solved_topics.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+#
+class CopySolvedTopicCustomFieldToDiscourseSolvedSolvedTopics < ActiveRecord::Migration[7.2]
```

Comment:
I think a migration spec will be helpful here to give us the confidence to know that this migration works.